homework.css

```diff
@@ -1,4 +1,10 @@
+:root {
+    --button-violet-color: rgb(61, 6, 113);
```

Comment:
A változók nevében értéket jelezni (`violet`) lehetnek kellemetlen hatásai a jövőre nézve (arculatváltásnál nem elég az értékét változtatni, a nevét is kell - néha 300 helyen).